src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -169,7 +170,12 @@ public boolean requiresToken() {
     public AwsCredentials resolveCredentials() {
 
         if (StringUtils.isBlank(iamRoleArn)) {
-            return AwsBasicCredentials.create(accessKey, secretKey.getPlainText());
+            if (StringUtils.isBlank(accessKey) && StringUtils.isBlank(secretKey.getPlainText())) {
+                // fall back to default credentials of node
+                return DefaultCredentialsProvider.builder().build().resolveCredentials();
```

Comment:
Does this work for your use case?

```suggestion
                return DefaultCredentialsProvider.create().resolveCredentials();
```

Reply:
Will test it

Reply:
yeah, that works